Lib/test/test_math.py

```diff
@@ -623,6 +624,94 @@ def testFmod(self):
         self.assertEqual(math.fmod(0.0, NINF), 0.0)
         self.assertRaises(ValueError, math.fmod, INF, INF)
 
+    def test_fmax(self):
+        self.assertRaises(TypeError, math.fmax)
+        self.assertRaises(TypeError, math.fmax, 'x', 'y')
+
+        self.assertEqual(math.fmax(0., 0.), 0.)
+        # fmax() does not need to be sensitive to the sign of 0 (F.10.9.2.3).
+        self.assertIn(math.fmax(0., -0.), {-0., 0.})
+        self.assertIn(math.fmax(-0., 0.), {-0., 0.})
+
+        self.assertEqual(math.fmax(1., 0.), 1.)
+        self.assertEqual(math.fmax(0., 1.), 1.)
+        self.assertEqual(math.fmax(1., -0.), 1.)
+        self.assertEqual(math.fmax(-0., 1.), 1.)
+
+        self.assertEqual(math.fmax(-1., 0.), 0.)
+        self.assertEqual(math.fmax(0., -1.), 0.)
+        self.assertEqual(math.fmax(-1., -0.), -0.)
+        self.assertEqual(math.fmax(-0., -1.), -0.)
```

Comment:
Again, such tests are insensitive to the zero sign.  Maybe you should rather pick some normal values to compare?

Reply:
Arf, I thought it was. Ok, I'll use better values.